src/visual.ts

```diff
@@ -92,6 +92,162 @@ module powerbi.extensibility.visual {
         private static Radians: number = Math.PI / 180;
         private static MinOpacity: number = 0.2;
         private static MaxOpacity: number = 1;
+
+
+        public static PreparedRandoms: number[] = [
+            0.7119651380581562,
+            0.9329344002304909,
+            0.662959468447071,
+            0.9228123970858741,
+            0.43461353653769996,
```

Comment:
Are we going to release it? If so, please increase version in `package.json` and `pbiviz.json`